src/index.js

```diff
@@ -16,10 +17,12 @@ module.exports = (web3) => {
       Scheduler,
       TxRequest,
       Util: Util(),
+      alarmClient
```

Comment:
Maybe make it start with an uppercase letter to match the rest?

Reply:
True that.